### ISSUE TITLE
Derivation read fails with chroot stores

### ISSUE THREAD

**Issue by lheckemann** [maralorn/nix-output-monitor#112]
#### What:
Lots of errors like
```
nix-output-monitor error: DerivationReadError /nix/store/d8sbf9afdpg05bak61lm1qyc3sjgqlq0-linux-config-6.1.51.drv: openFile: does not exist (No such file or directory)
```
#### How:
Run a build in a chroot store, e.g. `nom-build --no-substitute --store /tmp '<nixpkgs>' -A linux`

**Comment by maralorn** [comment]
Yeah, that is to be expected. I think the position of the store is even an assumption that I explicitly mention in the README.

I would be happy to have a solution to this but I can’t think of a good one.

**Comment by maralorn** [comment]
#158 might be a step in the right direction here.
